drafts/draft-ietf-ippm-ioam-data.xml

```diff
@@ -805,96 +805,18 @@ The trace option data MUST be 4-octet aligned:
               value and calculate the node length from the IOAM-Trace-Type
               bits.</t>
 
-              <t anchor="TraceFlags" hangText="Flags">4-bit field. The
-              following flags are defined: <list style="hanging">
+              <t anchor="TraceFlags" hangText="Flags">4-bit field. Flags
+			  are allocated by IANA, as specified in 
+			  <xref target="Flags-Registry-Sec"></xref>.
+			  The current document allocates a single flag as follows: 
+			  <list style="hanging">
                   <t hangText="Bit 0">"Overflow" (O-bit) (most significant
                   bit). This bit is set by the network element if there is not
                   enough number of octets left to record node data, no field
```

Comment:
s/if there is not enough number of octets left/if there are not enough octets left/